crates/re_query/Cargo.toml

```diff
@@ -29,6 +29,7 @@ re_arrow_store.workspace = true
 re_data_store.workspace = true
 re_format.workspace = true
 re_log_types.workspace = true
+re_types_core.workspace = true
```

Comment:
(here and elsewhere:) To clarify, we need both `re_types_core` and `re_types` because making them `re_types_core`-only _started_ but hasn't finished yet, correct?

Reply:
yes